assets/queries/terraform/alicloud/rds_instance_log_disconenctions_disabled/metadata.json

```
@@ -0,0 +1,11 @@
+{
+    "id": "d53f4123-f8d8-4224-8cb3-f920b151cc98",
+    "queryName": "RDS Instance Log Disconnections Disabled",
+    "severity": "LOW",
+    "category": "Observability",
+    "descriptionText": "log_disconnections parameter is not set to ON for RDS instances",
```

```suggestion
    "descriptionText": "log_disconnections parameter should be set to ON for RDS instances",
```

Can you please update in #4938?